fuzz/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+
+[package]
+name = "url-fuzz"
+version = "0.0.1"
+authors = ["Automatically generated"]
+
+[dependencies.url]
+path = ".."
+
+[[bin]]
+name = "parse"
+path = "fuzzers/parse.rs"
+
+[workspace]
+members = ["."]
```

Comment:
Instead of creating a new workspace here, this crate should be added to the workspace at the root of this repository.